sharktank/requirements.txt

```diff
@@ -12,3 +12,6 @@ datasets
 # Serving deps.
 fastapi>=0.112.2
 uvicorn>=0.30.6
+
+azure-identity>=1.19
+azure-storage-blob>=12.24
```

Comment:
Let's put these new requirements in a dev requirements file or an "extras" so they don't spill into the packages we publish unconditionally.

Reply:
The sharktank package currently has two requirements files, one for the standard install and one for the `sharktank[testing]` extra: https://github.com/nod-ai/shark-ai/blob/5e3f5e645abafc58c69a7f871e4b18b318d6653d/sharktank/pyproject.toml#L33-L37

Maybe `testing` is a good fit for this? We can also add a new category as needed

Reply:
This is not required by any tests right now. I added a dev package component. And put them in another file requirements file.